src/Generator.hpp

```diff
@@ -0,0 +1,5 @@
+#pragma once
+#include "Board.hpp"
+
+Lines getAllVectorPermutation_(int vSize);
```

Comment:
function is for development use only, won't have any use in final project

Reply:
put it in an unnamed namespace then.

Reply:
when i put it inside namespace {...} it is found to be undefined in tests

Reply:
In fact it you want to disallow others to use it, because it is "your internal development function" it should be tested like a private method - via some other public method call. You will need to change tests in future. Right now it can stay here but at least put it in some named namespace like `detail` or something else.

Reply:
I think i will delete it later after US4.3. It was for test purpose to learn to permute vectors with lexical table shorter then vector itself.